packages/pieces/community/mautic/src/lib/triggers/index.ts

```diff
@@ -0,0 +1,453 @@
+          "label": "Multiple Contact",
+          "alias": "multiple_contact",
+          "type": "multiselect",
+          "group": "core",
+          "object": "lead",
+          "is_fixed": "0",
+          "value": null
+        }
+      },
+      "social": {
+        "facebook": {
+          "id": "21",
+          "label": "Facebook",
+          "alias": "facebook",
+          "type": "text",
+          "group": "social",
+          "object": "lead",
+          "is_fixed": "0",
+          "value": null
+        },
+        "foursquare": {
+          "id": "22",
+          "label": "Foursquare",
+          "alias": "foursquare",
+          "type": "text",
+          "group": "social",
+          "object": "lead",
+          "is_fixed": "0",
+          "value": null
+        },
+        "instagram": {
+          "id": "24",
+          "label": "Instagram",
+          "alias": "instagram",
+          "type": "text",
+          "group": "social",
+          "object": "lead",
+          "is_fixed": "0",
+          "value": null
+        },
+        "linkedin": {
+          "id": "25",
+          "label": "LinkedIn",
+          "alias": "linkedin",
+          "type": "text",
+          "group": "social",
+          "object": "lead",
+          "is_fixed": "0",
+          "value": null
+        },
+        "skype": {
+          "id": "26",
+          "label": "Skype",
+          "alias": "skype",
+          "type": "text",
+          "group": "social",
+          "object": "lead",
+          "is_fixed": "0",
+          "value": null
+        },
+        "twitter": {
+          "id": "27",
+          "label": "Twitter",
+          "alias": "twitter",
+          "type": "text",
+          "group": "social",
+          "object": "lead",
+          "is_fixed": "0",
+          "value": null
+        }
+      },
+      "personal": [],
+      "professional": []
+    }
+  },
+  "channel": "email",
+  "old_status": "contactable",
+  "new_status": "manual",
+  "timestamp": "2017-12-01T00:05:18-06:00"
+}
+
+export const triggers = [
+  {
+    name: "lead_company_change",
+    displayName: "Contact Updated",
+    description: "Triggered when a contact is updated.",
+    sampleData: {
+      "mautic.lead_company_change": [contactTestData]
+    },
+    eventType: "mautic.lead_company_change",
+  },
+  {
+    name: "lead_post_save_update",
+    displayName: "Contact Company Subscription Change",
+    description: "Triggered when a commpany is added or removed to/from contact",
+    sampleData: {
+      "mautic.lead_post_save_update": [contactTestData]
+    },
+    eventType: "mautic.lead_post_save_update",
+  },
+  {
+    name: "lead_channel_subscription_changed",
+    displayName: "Contact Channel Subscription Change",
+    description: "Triggered when a contact's channel subscription status changes.",
+    sampleData: {
+      "mautic.lead_channel_subscription_changed": [contactTestData]
+    },
+    eventType: "mautic.lead_channel_subscription_changed",
```

Comment:
can you please give a reference for this eventType? I couldn't find it in REST API docs.

Reply:
You can get this by making a request to "<api>/hooks/triggers". Its available on make.com. The client funding, is interested in it.

Reply:
Most likely hidden from documents, dont know why

Reply:
Here is an example of the response from /hooks/triggers
```json
{
  "status": 200,
  "headers": {
    "server": "nginx",
    "date": "Thu, 25 Jan 2024 08:19:53 GMT",
    "content-type": "application/json",
    "transfer-encoding": "chunked",
    "connection": "close",
    "vary": "Accept-Encoding, Authorization",
    "mautic-version": "5.0.2",
    "cache-control": "no-cache, private"
  },
  "body": {
    "triggers": {
      "mautic.company_post_save": {
        "label": "Company Create/Update Event",
        "description": "Triggered when a company is created/updated"
      },
      "mautic.company_post_delete": {
        "label": "Company Deleted Event",
        "description": "Triggered when a company is deleted"
      },
      "mautic.lead_channel_subscription_changed": {
        "label": "Contact Channel Subscription Change Event",
        "description": "Triggered when a contact's channel subscription status changes."
      },
      "mautic.lead_company_change": {
        "label": "Contact Company Subscription Change Event",
        "description": "Triggered when a commpany is added or removed to/from contact"
      },
      "mautic.lead_post_delete": {
        "label": "Contact Deleted Event",
        "description": "Triggered when a contact is deleted."
      },
      "mautic.lead_post_save_new": {
        "label": "Contact Identified Event",
        "description": "Triggered when a contact is identified."
      },
      "mautic.lead_points_change": {
        "label": "Contact Points Changed Event",
        "description": "Triggered when a contact's points are modified."
      },
      "mautic.lead_list_change": {
        "label": "Contact Segment Membership Change Event",
        "description": "Triggered when a contact segment membership is changed"
      },
      "mautic.lead_post_save_update": {
        "label": "Contact Updated Event",
        "description": "Triggered when a contact is updated."
      },
      "mautic.email_on_open": {
        "label": "Email Open Event",
        "description": "mautic.email.webhook.event.open_desc"
      },
      "mautic.email_on_send": {
        "label": "Email Send Event",
        "description": "mautic.email.webhook.event.send_desc"
      },
      "mautic.form_on_submit": {
        "label": "Form Submit Event",
        "description": "mautic.form.webhook.event.form.submit_desc"
      },
      "mautic.page_on_hit": {
        "label": "Page Hit Event",
        "description": "mautic.page.webhook.event.hit_desc"
      },
      "mautic.sms_on_send": {
        "label": "Text Send Event",
        "description": "mautic.sms.webhook.event.send_desc"
      }
    }
  }
}
```

Reply:
@kanarelo Thanks for the details.